specification/api-tracing.md

```diff
@@ -70,6 +69,20 @@ A duration is the elapsed time between two events.
 * The minimal precision is milliseconds.
 * The maximal precision is nanoseconds.
 
+## Context Propagation
+
+OpenTelemetry separates its design into two layers: the top layer contains
+independent observability concerns, such as traces, metrics and correlation
+context. All concerns share an underlying distributed context propagation layer,
+for storing state and accessing data across the lifespan of a distributed
+transaction.
+
+Concerns access their data in-process using the same, shared context object.
+Each concern uses its own namespaced set of keys in the context, containing
```

Comment:
Same here: I don't think it matters to the user how the concerns work with the context, unless you're writing a new concern. Let's call this detail the SDK specification.